proposals/lip-0022.md

```diff
@@ -177,25 +177,17 @@ Internally, the algorithm works as follows:
 
 In the case that `ListStandbyDelegates` contains less than 2 elements, the delegates with rank 102 and 103 will be chosen as `delegate1 `and `delegate2` respectively.
 
-### Round Computation and Delegate List Ordering
+### Round Computation and Forger List
 
-The round computation is performed in the same way as in the current implementation with the difference that `delegate1` and `delegate2`, as defined in the previous section, are included in the forging delegate list of `round` + 1. This implies the following:
+A round is composed of 103 blocks now.
 
-*   The round length is now 103 block slots. Currently, this is defined by the constant  <code>[ACTIVE_DELEGATES](https://github.com/LiskHQ/lisk-sdk/blob/310283408102ff9f891830225afe8124e833a608/config/default/constants.js#L55)</code>.  For clarity, this constant should be replaced for three constants defining the number of the top 101 delegates by `delegateWeight`, the number of standby delegates, i.e., 2, and the length of the round (sum of the two previous constants).
-*   The change in the length of the round affects the logic to check when a round is finished which is 103 block slots in total.
-*   `delegate1` and `delegate2` have to be included in the delegate list<sup>3</sup> before it is re-ordered for `round` + 1. Currently, the function <code>[getKeysSortByVote](https://github.com/LiskHQ/lisk-sdk/blob/7c31c9d4b12a72f18a35ead79198b2ba6d4bd091/framework/src/modules/chain/submodules/delegates.js#L32)</code> computes this list before being re-ordered.
+Before being [shuffled](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0003.md), the forger list<sup>3</sup> for `round` + 1 is constructed as follows:
```

Comment:
```suggestion
Before being ordered according to [LIP 0003](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0003.md), the set of 103 forgers for `round + 1` is constructed as follows:
```

Reply:
For me it would be clearer if there is only one `forger list` for a round, i.e., the forgers in the order of assigned block slots. What is defined here is rather a set, as the order does not matter for the protocol (only the order in LIP 0003 matters). I find it a bit confusing to call this set forger list and also impose some arbitrary order on it.

Reply:
I also find the foot note confusing as here you specify how this list is computed and it is unclear what computing with a delay of 2 rounds means.

Reply:
You could also define the creation of the forger list here and make the 3rd step, order the delegates according to LIP 0003.

Reply:
Modified the first part as suggested. Also changed the footnote formulation to make it clearer, I dont want to remove it to have a quick link to check where this 2 rounds delay comes from.

Reply:
I also removed every occurrence of forging list as the structure being specified here.